pages/index.tsx

```diff
@@ -13,13 +14,34 @@ export default () => (
       <title>UNIHACK — Australia's premier student hackathon</title>
     </Head>
     <Page featureBackground>
-      <Hero/>
-      <EventDescription/>
+      <Hero />
+      <EventDescription />
+      <Faqs />
       <section className="cities">
-        <CityBanner city="Melbourne" year="2019" href="https://unihack2019.devpost.com/"/>
-        <CityBanner city="Sydney" year="2018" href="https://unihack2018syd.devpost.com/" left/>
+        <CityBanner
+          city="Melbourne"
+          year="2019"
+          href="https://unihack2019.devpost.com/"
+        />
+        <CityBanner
+          city="Sydney"
+          year="2018"
+          href="https://unihack2018syd.devpost.com/"
+          left
+        />
       </section>
-      <Sponsor/>
+      <Sponsor />
     </Page>
+    <a
```

Comment:
I'm thinking maybe this should go inside the hero component? also should it follow the page margins?

Reply:
Yeah, that works. Probably align it to the left so it works on mobile & desktop.